lib/sequelize.js

```diff
@@ -145,6 +149,7 @@ var Sequelize = function(database, username, password, options) {
     pool: {},
     quoteIdentifiers: true,
     hooks: {},
+    retry: {max: 5, match: ['SQLITE_BUSY: database is locked']},
```

Comment:
Yes i'm aware, but in the future would we apply the global defaults to both managed transactions and queries?